scripts/valhalla_build_config

```diff
@@ -270,6 +270,7 @@ config = {
         'max_exclude_polygons_length': 10000,
         'max_distance_disable_hierarchy_culling': 0,
     },
+    'baldr': {'live_speed_fading_sec': 3600},
```

Comment:
I'd actually put this into `thor` instead of `baldr`, which is actually only hosting data structures and thor is the only one really using this